consensus/istanbul/core/roundchange.go

```diff
@@ -175,12 +145,7 @@ func (c *core) handleRoundChangeCertificate(proposal istanbul.Subject, roundChan
 func (c *core) handleRoundChange(msg *istanbul.Message) error {
 	logger := c.newLogger("func", "handleRoundChange", "tag", "handleMsg", "from", msg.Address)
 
-	// Decode ROUND CHANGE message
-	var rc *istanbul.RoundChange
```

Comment:
Ditto that I'd like some form of this check (either enforced through the type system or check that `rc != nil`).